rust/rsc/src/rsc/blob.rs

```diff
@@ -0,0 +1,71 @@
+use crate::config::RSCConfig;
+use crate::types::GetUploadUrlResponse;
+use axum::{extract::Multipart, Json};
+use sea_orm::DatabaseConnection;
+use std::sync::Arc;
+use tracing;
+
+#[tracing::instrument]
+pub async fn get_upload_url(
+    _conn: Arc<DatabaseConnection>,
+    config: Arc<RSCConfig>,
+) -> Json<GetUploadUrlResponse> {
+    let url = config.server_addr.clone() + "/blob";
+    Json(GetUploadUrlResponse { url })
+}
+
+pub async fn create_blob(mut multipart: Multipart, config: Arc<RSCConfig>) -> hyper::StatusCode {
```

Comment:
Instead of passing the whole config, just pass the part that matters

Reply:
this thread was mixed up with the other one. For create_blob we actually do need a DB connection and more context about the config so I'm going to leave this in for now. Can circle back to trim it down after we have blob tracking